app/models/student.rb

```diff
@@ -67,6 +67,6 @@ def adjusted_grade_for_assignment(assignment_object)
   end
 
   def turned_in_assignment?(assignment_object)
-    self.grade_for_assignment(assignment_object) != 0
+    self.grade_for_assignment(@assignment) != 0
   end
 end
```

Comment:
The places where you changed `assignment_object` to `@assignment` in this file should be changed back.

`assignment_object` is valid–it's referring to the parameter for each method.